doc/source/whatsnew/v0.23.0.txt

```diff
@@ -898,6 +898,48 @@ yourself. To revert to the old setting, you can run this line:
 
   pd.options.display.max_columns = 20
 
+.. _whatsnew_0230.boxplot.sharexy:
+
+Optional sharing of x/y-axis by pandas.DataFrame().groupby().boxplot()
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(:issue:`15184`)
+
+Previous Behavior:
+
+.. code-block:: jupyter-notebook
+
+   %pylab inline
+    import pandas as pd
+
+    N = 100
+    rand = random.random(N)
+    clas = random.binomial(5,.5, N)
+    df = pd.DataFrame({'Rand': rand-clas,
+                    'Rand2': rand,
+                    'Class': clas},
+                    index= np.arange(N))
+
+    df.groupby('Class').boxplot(sharey=True, sharex=False)
+    >>> TypeError: boxplot() got an unexpected keyword argument 'sharey'
+
+    Using boxplot with keywords sharex or sharey resulted in an error.
+
+New Behavior:
+
+.. ipython:: jpyter-notebook:
+    
+    ...
+
+    df.groupby('Class').boxplot(sharey=True, sharex=True)
```

Comment:
Won't need all of these. Probably just one will convey the point.

Reply:
Ok,  0.23.1.txt does not exist yet. When I fetch from upstream I don't receive it. Does that mean I should create that file? I thought I already removed the example. Not sure why it still there.

Reply:
@soerendip did you merge after fetching? The file is there already on master